hca/util/__init__.py

```diff
@@ -107,6 +107,9 @@ def __init__(self, retry_after_status_codes={301}, **kwargs):
 
 
 class _ClientMethodFactory(object):
+    retry_policy = RetryPolicy(read=10, status=10, status_forcelist=frozenset({502, 503, 504}))
```

Comment:
Do you want to add 500? c.f., https://github.com/HumanCellAtlas/dcp-preview/issues/10

Reply:
Great catch. Added.